absl/hash/internal/hash.h

```diff
@@ -1136,7 +1136,7 @@ class ABSL_DLL MixingHashState : public HashStateBase<MixingHashState> {
   // probably per-build and not per-process.
   ABSL_ATTRIBUTE_ALWAYS_INLINE static uint64_t Seed() {
 #if (!defined(__clang__) || __clang_major__ > 11) && \
-    !defined(__apple_build_version__)
+    (!defined(__apple_build_version__) || __apple_build_version__ >= 19558921) // Xcode 12
```

Comment:
It looks like this changes the code from "only do this on Xcode >= 12" to "don't do this on any version of Xcode". Is that intended?

Reply:
Other way around

Reply:
I must have been looking at the diff backwards. Thanks.